client/src/components/App/index.js

```diff
@@ -1,12 +1,97 @@
 import React from 'react';
+import {
+  BrowserRouter as Router,
```

Comment:
there is no need to call this here , you already called in the index file in the source .

Reply:
.